README.md

```diff
@@ -5,6 +5,7 @@ using MRI images from BrainLab.
 It can deal with brains with lesions and extensive atrophy and extracts the brain in seconds.
 It can use a T1-weighted image or a combination of T1, T2 and Flair scans as input. 
 
+We recommend using iCVMapp3r with the Docker or Singularity containers we provide, since support for local installs has been discontinued. See our doc: [installation instructions](https://github.com/AICONSlab/iCVMapp3r/blob/master/docs/docker.md) for more information.
```

Comment:
Shouldn't this link the to the official readthedocs entry instead of the code entry on GitHub? Please change this to `[installation instructions](https://icvmapp3r.readthedocs.io/en/latest/docker.html)`.

Reply:
The ANTs installation has been fixed in my last commit.